### DISCUSSION TITLE
ATOMSTACK M40 cpde

### DISCUSSION THREAD

**Discussion by JKDiehnel** [arkypita/LaserGRBL#1710] [Q&A]
What is the correct S-Max value to use in the LaserGRBL Configuration for an ATOMSTACK M40 Laser head

**Comment by Tribo-2dot0** [comment]
See the value of $30 in your Grbl settings.
![Captura de Tela (95)](https://user-images.githubusercontent.com/83982525/148301987-7380be9d-5ce8-4746-8150-78db664a6e5a.png)
